assets/jsons/changelogs.json

```diff
@@ -0,0 +1,14 @@
+{
+  "27.1.3" : {
+    "title" : "What's News ?",
+    "htmlBody" : "<div style='max-width: 800px; margin: auto; font-family: Arial, sans-serif;'> <h3 style='color: #FF5733; font-size: 1.2em; font-weight: bold; margin-bottom: 0.5em;'>Features</h3> <ul style='list-style-type: disc; padding-left: 20px; margin-bottom: 1em;'> <li>Beat Saber versions now automatically sync with LIV #320</li> <li>Use ArcViewer to preview maps #312</li> </ul> <h3 style='color: #33C1FF; font-size: 1.2em; font-weight: bold; margin-bottom: 0.5em;'>Fixes</h3> <ul style='list-style-type: disc; padding-left: 20px; margin-bottom: 1em;'> <li>SteamVR backup folder is now restored when disabling FPFC Mode #314</li> <li>OneClick installs now correctly place contents in user-defined symlinks #311</li> <li>Currently downloading Beat Saber version no longer disappears on uninstalling a version #307</li> <li>Add 'no-yeet' argument when launching Beat Saber to prevent mods to be removed if 'UserData' is linked #321</li> <li>Fixed a rare bug that prevent BSManager from launching due to a missing DLL #328</li> <li>Increase the SteamVR restore timeout to prevent it from launching in FPFC Mode #330</li> </ul> <h3 style='color: #4CD137; font-size: 1.2em; font-weight: bold; margin-bottom: 0.5em;'>Other changes</h3> <ul style='list-style-type: disc; padding-left: 20px; margin-bottom: 1em;'> <li>Improved Spanish translation #309 (Thanks to @GabiRP)</li> <li>For new installations, the default folder for BSManager contents has been moved from 'Documents' to the 'User' directory to avoid OneDrive conflicts #319</li> </ul> </div> ",
+    "timestampPublished" : "1700788913",
```

Comment:
i don't like the name of this property... `timestamp` isn't enought ? 'Caus `Published` is already inherant to the contents of this json